sdk/program/src/bpf_loader_upgradeable.rs

```diff
@@ -24,6 +24,9 @@ use crate::{
 
 crate::declare_id!("BPFLoaderUpgradeab1e11111111111111111111111");
 
+/// Cooldown before a program can be redeployed again
+pub const DEPLOYMENT_COOLDOWN_IN_SLOTS: u64 = 750;
```

Comment:
If people want to quickly redeploy their program often on a test validator, would they have to wait 5 minutes between deployments or reset the ledger?  It would be nice to have this value 0 for test environments, otherwise I imagine we'll get complaints from dev who want to keep up their ledger as they iterate.

I can't come up with a nice way to do this though, other than by adding a *separate* feature that sets this to 0, which never gets turned on, but is enabled in test environments since all features are turned on there. What do you think?

Reply:
Ah ok, fine by me, as long as you get the needed functionality!